Dockerfile

```diff
@@ -1,18 +1,26 @@
-# Take advantage of a multi-stage build to avoid dev-dependencies in the final image
-FROM node:16-alpine AS builder
+FROM node:16-alpine AS build
+ENV NODE_ENV=development
 
 WORKDIR /app
 
 COPY . .
 
-RUN yarn
+RUN yarn install
 RUN yarn build
 
-FROM node:16-alpine
+FROM node:16-slim AS dist
```

Comment:
Why not stick with alpine?

Reply:
I thought `slim` is smaller, but it was a mistake